core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -324,8 +328,23 @@ protected void doStop() throws Exception {
     StatisticsHandler statsHandler = new StatisticsHandler();
     statsHandler.setHandler(handlers);
 
-    server.setHandler(statsHandler);
-
+    final ServletContextHandler webSocketServletContext =
+        new ServletContextHandler(ServletContextHandler.SESSIONS);
+    webSocketServletContext.setContextPath("/ws");
```

Comment:
should maybe make this configurable?

Reply:
maybe? is this Path part of the URL path or some other terminology? i've forgotten how jetty actually works. if it is user-facing, then I would say yes, it should be configurable.

Reply:
yeah, its part of the path. this would collide with anyone who used a path prefix of `/ws` previously (nobody afaik)